src/serlio/utils/MELScriptBuilder.cpp

```diff
@@ -160,6 +160,14 @@ void MELScriptBuilder::getWorkspaceDir() {
 	commandStream << L"workspace -q -rd;\n";
 }
 
+void MELScriptBuilder::getStringFromURL(const std::wstring& url) {
+	python(L"import urllib.request");
+	python(L"url = \\\"" + url + L"\\\"");
+	python(L"response = urllib.request.urlopen(url)");

```

Comment:
Did you consider using C++ instead of Python for the file download?
There are moderately easy-to-integrate C++ libraries to read URLs, like https://github.com/libcpr/cpr

Reply:
I considered it, but wanted to avoid additional using external dependencies that are not standard or handled by us.